### PR TITLE
update_attributes has been deprecated

### PR DESCRIPTION
**Pull request by d-m-u**
```update_attributes``` was replaced by ```update``` and this needed to happen sometime

### CODE REVIEW

**Comment by gmcculloug** [comment]
To clarify `update_attributes` and `update_attributes!` are deprecated starting in Rails 6.
See https://rubyinrails.com/2019/04/09/rails-6-1-activerecord-deprecates-update-attributes-methods/ for details and Rails PR links.

File: app/models/miq_ae_yaml_import.rb [hunk 1]
```diff
@@ -310,10 +310,10 @@ def new_domain_name_valid?
     true
   end
 
-  def update_attributes(domain_obj)
+  def update(domain_obj)
```
Comment (gmcculloug) [inline app/models/miq_ae_yaml_import.rb]: All these changes are pretty straightforward replacements.  Changing this method name is the only one that has potentially more exposure.

@mkanoor Do you know if this method would be called from any other repo where we might need to make the corresponding name change?

Comment (d-m-u) [reply app/models/miq_ae_yaml_import.rb]: Oh that was me being an idiot, sorry, it was late. This shouldn't be here.

Comment (gmcculloug) [reply app/models/miq_ae_yaml_import.rb]: Maybe/maybe not.  It is likely this name was picked to match the Rails method, so changing it to match may be the right thing.  Let's give @mkanoor  some time to respond before changing.

File: app/models/miq_ae_yaml_import.rb [hunk 2]
```diff
@@ -310,10 +310,10 @@ def new_domain_name_valid?
...
     return if domain_obj.name.downcase == MiqAeDatastore::MANAGEIQ_DOMAIN.downcase
     attrs = @options.slice('enabled', 'source')
-    domain_obj.update_attributes(attrs) unless attrs.empty?
+    domain_obj.update(attrs) unless attrs.empty?
```
Comment (mkanoor) [inline app/models/miq_ae_yaml_import.rb]: 👍

File: app/models/miq_ae_yaml_import.rb (hunk 1 above)
Comment (gmcculloug) [reply app/models/miq_ae_yaml_import.rb]: Confirmed that https://github.com/ManageIQ/manageiq-automation_engine/pull/367#pullrequestreview-288041339 was in response to my question here.  So we are good with the method name change.

Thanks @d-m-u

**Pull request merged**